server/grammar/YmlParser.g4

```diff
@@ -150,7 +150,7 @@ returnField:
 
 classPropertiesBlock: CLASSPROPERTIES classOption=field*;
 
-documentation: DOCUMENTATION;
+documentation: TRIPLE_QUOTE ANY* TRIPLE_QUOTE;
```

Comment:
Is it possible to have a documentation with single quote string ? Or is it only for the fold so it is not relevant here ?

Reply:
Yes, that's possible. In fact, I should probably rename this rule to `multilineString`.
It's named like this because a long time ago I believed that multiline strings were accepted by YE only with this documentation attribute :sweat_smile: